### PR TITLE
Port Tutor page to React

### PR DESCRIPTION
**Pull request by RoyEJohnson**
Issue #1791

### CODE REVIEW

File: src/app/pages/openstax-tutor/how-it-works.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {RawHTML} from '~/components/jsx-helpers/jsx-helpers.jsx';
+
+export default function HowItWorks({model}) {
+    const blurbs = [1, 2, 3, 4].map((num) => (
```
Comment (nathanstitt) [inline src/app/pages/openstax-tutor/how-it-works.js:5]: lodash has a handy `range` method for stuff like this https://lodash.com/docs/4.17.15#range

File: src/app/pages/openstax-tutor/learn-more.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                <div>{model.section7Subheading}</div>
+                <div class="buttons-row">
+                    {
+                        [1, 2].map((num) => <ButtonBox num={num} key={num} model={model} />)
+                    }
+                </div>
+            </div>
+        </section>
+    );
+}
```
Comment (nathanstitt) [inline src/app/pages/openstax-tutor/learn-more.js:33]: just curious but what is the `section7` that's mentioned several places in this page?

File: test/src/pages/openstax-tutor/openstax-tutor.test.js [hunk 3]
```diff
@@ -1,19 +1,32 @@
...
+            faqs: [
+                {
+                    "type": "faq",
+                    "value": {
+                        "question": "<p>How much time does it take to set up my course in OpenStax Tutor Beta?</p>",
+                        "slug": "how-much-time-to-set-up-course",
+                        "answer": "<p>It takes 10-20 minutes to set up your OpenStax Tutor Beta account and create your course within the tool once you have your verified instructor account. Detailed page tips walk you through each feature of the tool step by step to help you hit the ground running. We manually verify every instructor account to make sure students don’t get access to instructor materials, so if you don’t yet have an instructor account with OpenStax, expect to wait 3-5 days for our Support Team to verify you.</p>",
+                        "document": null
+                    },
+                    "id": "fb6cdaf3-0045-4392-a686-d809452bc2bd"
```
Comment (nathanstitt) [inline test/src/pages/openstax-tutor/openstax-tutor.test.js:18]: I know you're just porting, not rewriting, but verification should be instantaneous now with SureId, right?

File: src/app/pages/openstax-tutor/learn-more.js (hunk 2 above)
Comment (RoyEJohnson) [reply src/app/pages/openstax-tutor/learn-more.js:33]: In the CMS the data for each section is grouped by naming the data items with a prefix. The data for the Learn More section is prefixed section7. The FAQ is section6, etc.

File: test/src/pages/openstax-tutor/openstax-tutor.test.js (hunk 3 above)
Comment (RoyEJohnson) [reply test/src/pages/openstax-tutor/openstax-tutor.test.js:18]: This is only test data. The actual data is maintained in the CMS.

**Pull request merged**